src/main/java/frc/robot/commands/ShootingPosCommands/PodiumPosition.java

```diff
@@ -2,21 +2,25 @@
 
 import edu.wpi.first.wpilibj2.command.Command;
 import frc.robot.subsystems.ElevatorSubsystem;
+import frc.robot.subsystems.FlyWheelSubsystem;
 import frc.robot.subsystems.PivotSubsystem;
 
 public class PodiumPosition extends Command {
-    PivotSubsystem pivot;
-    ElevatorSubsystem elevator;
+    private final PivotSubsystem pivot;
+    private final ElevatorSubsystem elevator;
+    private final FlyWheelSubsystem flywheel;
 
-    public PodiumPosition(PivotSubsystem pivot, ElevatorSubsystem elevator){
+    public PodiumPosition(PivotSubsystem pivot, ElevatorSubsystem elevator, FlyWheelSubsystem flywheel){
         this.pivot = pivot;
         this.elevator = elevator;
+        this.flywheel = flywheel;
     }
 
     @Override
     public void execute(){
         pivot.pivotPodiumPosition();
         elevator.podiumPosition();
+        flywheel.podiumShot();
```

Comment:
Same comment about ramping this down when the command ends.

Also should pivot/elevator return to base position after this?

Reply:
Yes, the pivot and elevator should return to base pos, will update.

I'm going to look in the code to see about the missing ramp down statements.